PurchasesCoreSwift/Caching/DeviceCache.swift

```diff
@@ -38,7 +38,7 @@ import Foundation
     public init(userDefaults: UserDefaults = UserDefaults.standard,
                 offeringsCachedObject: InMemoryCachedObject<Offerings>? = InMemoryCachedObject(),
                 notificationCenter: NotificationCenter? = NotificationCenter.default,
-                assertionFunction: @escaping (String) -> Void = { assertionFailure($0) }) {
+                assertionFunction: @escaping (String) -> Void = { fatalError($0) }) {
```

Comment:
I changed this because during `release` builds, `assertionFailure` turns into a no-op.

Reply:
👍